### ISSUE TITLE
Make setup_op_mainnet_without_ovm general purpose

### ISSUE THREAD

**Issue by mattsse** [paradigmxyz/reth#12090]
### Describe the feature

This fills in dummy blocks:
https://github.com/paradigmxyz/reth/blob/fa59bd512e877e4677890b5c154b80816c07480e/crates/optimism/cli/src/commands/init_state/bedrock.rs#L12-L20

which can be useful for other chains as well https://github.com/paradigmxyz/reth/discussions/12057

currently this has hardcoded bedrock constants

## TODO
* refactor setup_op_mainnet_without_ovm so that it's general purpose

basically all of these constants should be an argument instead, or we transform all of this into a helper type instead
https://github.com/paradigmxyz/reth/blob/fa59bd512e877e4677890b5c154b80816c07480e/crates/optimism/cli/src/commands/init_state/bedrock.rs#L43-L43

cc @debjit-bw

**Comment by lean-apple** [comment]
Can I work on it ?

**Comment by hoank101** [comment]
hey sir, can i do it?

**Comment by mattsse** [comment]
assigned

**Comment by debjit-bw** [comment]
thanks a lot :p

**Comment by debjit-bw** [comment]
Hey guys, unfortunately I need this particular feature urgently. @hoank101 if it would take you more than today, would you mind if I please pick this up?

**Comment by debjit-bw** [comment]
Hey @mattsse, I just raised the PR for this #12134
